examples/cpp/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ project(example-cpp)
 set(CMAKE_CXX_STANDARD 20)
 
 include_directories(../../src/include)
+include_directories(../../third_party/nlohmann_json)
```

Comment:
Can you double check if we need this third party any more? I think we removed this dependency since #1751

Reply:
No, it's no longer needed. The example now works without any modifications.